### ISSUE TITLE
Blood Cell Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#36]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Blood Cell Detection</br>
:red_circle: **Aim** : Create a model which will detect the blood cells.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/adhoppin/blood-cell-detection-datatset </br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by sid-im** [comment]
plz assign me this project @abhisheks008

**Comment by LahariBoni** [comment]
Assigned it to you @sid-im! Go Ahead.

**Comment by abhisheks008** [comment]
Can you share me the approach you are going to take while solving the issue. And please mention the required credentials written in the issue. @sid-im

**Comment by LahariBoni** [comment]
> Can you share me the approach you are going to take while solving the issue. And please mention the required credentials written in the issue. @sid-im 

Yes @sid-im, It'd be great if you can kindly share your approach to work on this issue🙂

**Comment by sid-im** [comment]
Full name : Siddarth Akkatangerhal
GitHub Profile Link :  https://github.com/sid-im
Email ID : siddartha2510@gmail.com
Participant ID (if applicable): 
Approach for this Project :
i will use TensorFlow/Keras convolutional neural network to make our predictions using given data set.

**Comment by abhisheks008** [comment]
@sid-im updates please.

**Comment by khushaallll** [comment]
Full name : Khushal Koli
    GitHub Profile Link : https://github.com/khushaallll
    Email ID : kolikhki19it@student.mes.ac.in
    Participant ID (if applicable): 
    Approach for this Project : 

- Planning to use different CNN models.
- Planning on to use few pre-trained models, and YOLOv5
- will try to construct a custom neural network as well 

    What is your participant role : -

**Comment by abhisheks008** [comment]
Are you a participant from any open source program or, just wanna contribute here as a contributor?
@khushaallll

**Comment by khushaallll** [comment]
No I'm not a participant from open source program I just want to contribute as an independent contributor

**Comment by abhisheks008** [comment]
Sounds great. Issue has been assigned to you @khushaallll. Go ahead!

**Comment by abhisheks008** [comment]
Hey @khushaallll updates regarding this issue.

**Comment by khushaallll** [comment]
> Hey @khushaallll updates regarding this issue

Yes working on it. Will be done by this weekend

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@khushaallll
